### ISSUE TITLE
USB icons on switch controls

### ISSUE THREAD

**Issue by mindplay-dk** [ZorinOS/zorin-desktop-themes#6]
I've tried [asking](https://zoringroup.com/forum/viewtopic.php?f=5&t=15237) this question on the forums, and everyone who replied is as baffled as I am.

I installed this theme on a stock Ubuntu 19.10 by cloning this repository, copying the theme files to `/usr/share/themes`, and enabling it from `gnome-tweaks`.

On a Zorin installation, I'm seeing this:

![image](https://i.imgur.com/imRERdf.png)

But on my Ubuntu installation with the theme enabled, I'm seeing this instead:

![image](https://i.imgur.com/mjnJunv.png)

It appears to be a USB icon and a circle-x, instead of the checkmark and x that I'm seeing on a Zorin OS installation. My first thought was "wrong font", but if I look inside the [SVG file](https://github.com/ZorinOS/zorin-desktop-themes/blob/master/ZorinBlue-Light/gnome-shell/assets/toggle-on.svg) in the theme, these aren't even text, they're shapes - so how could they possibly get replaced with something else on my system?

This affects every switch control in every application.

Is there some sort of dependency I'm missing, like a font? And how does a font enter into the equation, since apparently these are just shapes in an SVG file?

Are the SVG files actually even being used, or are they just there to provide the source material for creating some other files that are actually part of the theme? What am I missing here? 🤔

**Comment by ArtyomZorin** [comment]
Thanks for the reports! The issue should be fixed in the new 2.1.3 release.

**Comment by mindplay-dk** [comment]
@ArtyomZorin Could you briefly say what the problem was? 🙂

I was trying to understand how themes work, and this part baffled me - where did these mysterious characters come from? They weren't present in the SVG files, and the icons in the SVG files were paths, not fonts, so it's impossible for me to understand how they could somehow turn into something else.

**Comment by ArtyomZorin** [comment]
By default Gtk draws an "I" character when the switch is on and "O" when it's switched off. In the Zorin themes we set the font to size zero in order to hide these characters, as we wanted to overlay the "✓" & "x"  asset on the switch instead.

This works fine in Gtk 3.22 (which is standard in Zorin OS 15), but the font size 0 property appears to have been ignored in newer versions of Gtk. As a result both the "I" & "O" characters as well as the "✓" & "x" asset are showing up on top of each other.

In order to work around this issue, we set the foreground colour in the switch (which was used by both the "I" & "O" characters as well as the "✓" & "x" asset) to transparent and removed the "✓" & "x" asset from the theme entirely. This workaround, in turn, also made the switches look a little cleaner in the new update (see picture below).
![switches](https://user-images.githubusercontent.com/3945334/77645066-03d12780-6f5a-11ea-8a0f-a07818510ec9.png)
